tests/unit/test_user.py

```diff
@@ -116,8 +116,7 @@ def test_update_class_method(self, create_join_staff, faker):
         UserModel.update(UserSchema, user.id, {"email": email})
         lookedup = UserModel.find(user.id)
         assert lookedup.email == email
-        assert lookedup.password is None
-        assert lookedup.hash_digest == user.hash_digest
+        assert lookedup.password == user.password
```

Comment:
I think we can delete this assertion. We're now no longer need to test behavior that we implemented. This is testing behavior provided by sqlalchemy.